pkg/fanal/analyzer/pkg/dpkg/dpkg.go

```diff
@@ -3,25 +3,28 @@ package dpkg
 import (
 	"bufio"
 	"context"
-	"log"
+	"fmt"
 	"os"
 	"path/filepath"
 	"regexp"
+	"sort"
 	"strings"
 
 	debVersion "github.com/knqyf263/go-deb-version"
+	"github.com/samber/lo"
 	"golang.org/x/xerrors"
 
 	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
 	"github.com/aquasecurity/trivy/pkg/fanal/types"
+	"github.com/aquasecurity/trivy/pkg/log"
 )
 
 func init() {
 	analyzer.RegisterAnalyzer(&dpkgAnalyzer{})
 }
 
 const (
-	version = 2
+	analyzerVersion = 2
```

Comment:
Need to update the analyzer version. 

```suggestion
	analyzerVersion = 3
```